common/changes/office-ui-fabric-react/tagpicker-textbox_2018-10-03-22-09.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "TagPicker textbox is in the wrong position - middle instead of bottom. Added an align style to fix this.",
+      "type": "patch"
+    }
+  ],
+  "packageName": "office-ui-fabric-react",
+  "email": "aneeshack4@gmail.com"
```

Comment:
You might want to update your `git config user.email` so this pulls-in your work address